rotkehlchen/data_import/importers/nexo.py

```diff
@@ -128,6 +136,12 @@ def _consume_nexo(
                 log.debug(f'Ignoring nexo entry {csv_row} with negative interest')
                 return
 
+            # from the csv a user shared, some entries with type of
```

Comment:
this seems like a copy pasta comment from above. There is no output amount here?

Reply:
yeah, from the csv. output amount & output currency are always empty for such cases.